### PR TITLE
Theme: Dark🌙 and Light☀ Theme added

### PR DESCRIPTION
**Pull request by t007rushi**
Light and dark theme added with toggler button at top-right with icons

recording

<details><summary>Browser metadata</summary>

```
Path:      /notes
Browser:   Chrome 99.0.4844.84 on Windows 10
Viewport:  1536 x 775 @1.25x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-7--monkeep.netlify.app%2Fnotes&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x775&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-7--monkeep.netlify.app/notes) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6246a8a65275270008e02c7d&commentID=6246aa138cb09b4cf3be52dc&resolution=resolved)

### CODE REVIEW

**Review by Manju-sree — CHANGES_REQUESTED** [review]
UI looks cool with theme, please try to add proper description, i have noticed that when in light mode, in (Take a Note..) section words are not visible.

File: src/context/theme-context.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const LocalTheme = () => localStorage.getItem("note_theme");
+
+const ThemeProvider = ({ children }) => {
+  const [theme, setTheme] = useState(LocalTheme);
+
+  const toggleTheme = () =>
+    setTheme((theme) => (theme === "light" ? "dark" : "light"));
+
+  useLayoutEffect(() => {
```
Comment (Manju-sree) [inline src/context/theme-context.js:13]: it's new to me about using useLayoutEffect( ).good to see you have implemented it. but not sure you have used javascript in this.

File: src/App.css [hunk 2]
```diff
@@ -1,13 +1,56 @@
...
+  --note-color-3: #fff475;
+  --note-color-4: #ccff90;
+  --note-color-5: #a7ffeb;
+  --note-color-6: #cbf0f8;
+  --note-color-7: #aecbfa;
+  --note-color-8: #d7aefb;
+  --note-color-9: #fdcfe8;
+}
+
+.dark-theme {
```
Comment (Manju-sree) [inline src/App.css:33]: good to see clean code and you have put so much efforts in making dark and light theme.

**Review by JantuDeb — APPROVED** [review]
This is really awesome that you have used rich text editor.
You have some issues with UI like, lists numbers are overflowing the note card

**Review by Manju-sree — APPROVED** [review]
Good job !!, This PR is good to merge now.

File: src/context/theme-context.js (hunk 1 above)
Comment (t007rushi) [reply src/context/theme-context.js:13]: No, its hook same as useEffect just sync

**Pull request merged**